MobiFlight/MobiFlightModule.cs

```diff
@@ -910,16 +911,29 @@ public IModuleInfo GetInfo()
                     devInfo.Version = "1.0.0";
                 }
 
-                // With the support of Custom Devices
-                // we also introduced CoreVersion
                 if (InfoCommand.Arguments.Length > 4)
                 {
-                    CoreVersion = InfoCommand.ReadStringArg();
+                    devInfo.CoreVersion = InfoCommand.ReadStringArg();
+                }
+                else
+                {
+                    // With the support of Custom Devices
+                    // we also introduced CoreVersion
+                    // if a CoreVersion was not provided 
+                    // we determine a fallback version
+                    devInfo.CoreVersion = FallbackCoreVersion(devInfo.Version, devInfo.Board);
+                    // if no guess is possible, set CoreVersion to Version
+                    // this reflectes all standard boards with FW version below 2.5.0
+                    if (String.IsNullOrEmpty(devInfo.CoreVersion))
+                    {
+                        devInfo.CoreVersion = devInfo.Version;
+                    }

```

Comment:
Can this be moved inside FallbackCoreVersion, so it returns Version instead of null?

Reply:
Yeah, I was also thinking about this. I left it here as I am still thinking that the fallback function is not required. If I am wrong, I will move it.